extensions/Cake.Tyrannoport.yml

```diff
@@ -0,0 +1,21 @@
+Type: Addin
+Name: Cake.Tyrannoport
+NuGet: Cake.Tyrannoport
+Assemblies:
+- /**/Cake.Tyrannoport.dll
+Repository: https://github.com/iwillspeak/Tyrannoport.git
+ProjectUrl: https://www.nuget.org/packages/Cake.Tyrannoport/
+Author: iwillspeak
+Description: "\U0001F996 For processing TRX files into something more enjoyable \U0001F996"
```

Comment:
We should check if we can handle unicode characters in description. Otherwise we should strip them manually from the description